fem/bilininteg.hpp

```diff
@@ -43,6 +43,9 @@ class BilinearFormIntegrator : public NonlinearFormIntegrator
    /** The result of the partial assembly is stored internally so that it can be
        used later in the methods AddMultPA() and AddMultTransposePA(). */
    virtual void AssemblePA(const FiniteElementSpace &fes);
+   /** Used with BilinearFormIntegrators that have different spaces. */
+   virtual void AssemblePA(const FiniteElementSpace &trial_fes,
+                           const FiniteElementSpace &test_fes);
```

Comment:
While building the `next` branch with CUDA enabled, I noticed that this new overload of `AssemblePA` causes numerous warnings like this:
```console
fem/bilininteg.hpp(1671): warning: overloaded virtual function "mfem::BilinearFormIntegrator::AssemblePA" is only partially overridden in class "mfem::GradientIntegrator"
```
One way to fix this is to add `using BilinearFormIntegrator::AssemblePA;` to all subclasses. Another is to change the name of the method slightly, e.g. `MixedAssemblePA`.

Reply:
I see that there was a similar issue in #1159 that @camierjs fixed there by adding `using` statements. In that PR the same method as above was introduced at the level of the parent class, `NonlinearFormIntegrator`.

Reply:
I don't have a cuda compiler setup. Can someone who has make the appropriate changes?

Reply:
@camierjs, do you think you can help @jandrej with this?

Reply:
`done`

Reply:
Thanks!